tardis/plasma/base.py

```diff
@@ -14,11 +14,11 @@
 class BasePlasma(object):
 
     outputs_dict = {}
-    def __init__(self, plasma_properties, **kwargs):
+    def __init__(self, plasma_properties, property_args=None, **kwargs):
```

Comment:
@ftsamis that looks awesome! I'm wondering if we should call this `property_kwargs`? But if you think `args` is better let's go with this.